test/check-machines-storage-pools

```diff
@@ -488,6 +488,19 @@ class TestMachinesStoragePools(machineslib.VirtualMachinesCase):
             source={"source_path": "vol_grp1"},
         ).execute()
 
+        # Prepare a disk with ext4 filesystem to test the fs storage pool type
+        m.execute(f"parted {dev} mklabel gpt")
+        m.execute(f"parted -s {dev} mkpart primary ext4 0% 100%")
+        part_dev = f"{dev}p1"
+        m.execute(f"mkfs.ext4 {part_dev}")
+        StoragePoolCreateDialog(
+            self,
+            name="my_fs_pool",
+            pool_type="fs",
+            target="/media/",
+            source={"source_path": dev, "format":"ext4"},
```

Comment:
Shouldn't this be `part_dev`? The test goes through the trouble of creating a partition, but then doesn't actually use it, and instead gives the raw device here?

Reply:
Thank you, you're right, it's `part_dev` correctly.
Now that the test set works on hand, I'm fixing it along with other minor fixes.